pylint_odoo/checkers/no_modules.py

```diff
@@ -231,9 +231,6 @@
     'create', 'write', 'read', 'unlink', 'copy',
     'setUp', 'setUpClass', 'tearDown', 'default_get',
 ]
-DFTL_VALID_ODOO_VERSIONS = [
```

Comment:
Why you changed this global variable to `misc`?

Reply:
@moylop260 
I try to used the `DFTL_VALID_ODOO_VERSIONS` since `no_modules.py` but being inside the file `misc.py` I try to import the `DFTL_VALID_ODOO_VERSIONS` but this import fail, below the output

```python
(Pdb) from .checkers.no_modules import DFTL_VALID_ODOO_VERSIONS
*** ImportError: cannot import name 'DFTL_MANIFEST_DATA_KEYS'
```
This trouble is for the import `from .modules_odoo import DFTL_MANIFEST_DATA_KEYS`

For that reason I moved this definition